Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+# What is an Inclusive Minute
+
+An inclusive minute (or inclusive moment) is an inclusivity practice designed for teams to easily add to existing meeting agendas.
+The practice is intentionally kept small, informal and uncontroversial to maximize the likelihood of adoption and uptake.
+The purpose is to raise awareness, inspire dialog, deeper thinking and/or inform future action.
+Although rather modest in its aims, especially when compared with the need it is intended to address, it is also a useful approach in helping to seed changes in culture.
```

Comment:
Maybe this can be phrased differently. The sentence as-is reads like it's undermining the practice instead of recommending it. Suggestion:
```suggestion
Although rather modest in its aims, highlighting inclusivity as a topic regularly in meetings can sow seeds that can grow into larger cultural change.
```